src/components/ResultCount.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+    <Text text="{{ count }} movie(s) found" />
+</template>
+
+<script lang="ts">
+
+export default {
+    name: 'ResultCount',
+    components: [Text],

```

Comment:
need to import Text component
and use `components: { Text }` instead of `components: [Text]`

Reply:
@rnesteruk13 fixed